observations/atmosphere/aircraft.yaml.j2

```diff
@@ -7,146 +7,552 @@
     obsdatain:
       engine:
         type: H5File
-        obsfile: "{{atmosphere_obsdatain_path}}/{{atmosphere_obsdatain_prefix}}{{observation_from_jcb}}{{atmosphere_obsdatain_suffix}}"
+        #obsfile: "/work2/noaa/da/esposito/ufoeval/GSIobserver_NE_ADC/2021080100/ioda/obs/aircraft_obs_2021080100.nc4" #NICKE test
+        obsfile: "{{atmosphere_obsdatain_path}}/{{atmosphere_obsdatain_prefix}}prepbufr.acft_profiles"
       obsgrouping:
-        group variables: ["stationIdentification"]
-        sort variable: "pressure"
-        sort order: "descending"
+        group variables: [stationIdentification]
+        sort variable: pressure
+        sort order: descending
     obsdataout:
       engine:
         type: H5File
-        obsfile: "{{atmosphere_obsdataout_path}}/{{atmosphere_obsdataout_prefix}}{{observation_from_jcb}}{{atmosphere_obsdataout_suffix}}"
+        #obsfile: "/work2/noaa/da/nesposito/BufrConverterTest_aircraft/diags/2021080100/diags_aircft_prepbufr.2021080100.nc" #NICKE test
+        obsfile: "{{atmosphere_obsdataout_path}}/{{atmosphere_obsdataout_prefix}}prepbufr.acft_profiles_{{atmosphere_obsdataout_suffix}}"
     io pool:
       max pool size: 1
-    simulated variables: [windEastward, windNorthward, airTemperature, specificHumidity]
-  #
+    simulated variables: [airTemperature, windEastward, windNorthward, specificHumidity]
 
   # Observation Operator
   # --------------------
   obs operator:
-    name: VertInterp
-  #
+    name: Composite
+    components:
+    - name: VertInterp
+      variables:
+      - name: windEastward
+      - name: windNorthward
+      # Hofx scaling
+      hofx scaling field: SurfaceWindScalingPressure
+      hofx scaling field group: DerivedVariables
 
-  # Observation Filters (QC)
-  # ------------------------
-  obs filters:
-    # Observation Range Sanity Check
-  - filter: Bounds Check
+    - name: VertInterp
+      observation alias file: "/work2/noaa/da/acollard/git/GDASApp/parm/atm/obs/testing/swell/obsop_name_map.yaml"
+      variables: [airTemperature ]
+
+    - name: VertInterp
+      variables:
+      - name: specificHumidity 
+
+  # Linear Observation Operator
+  # ---------------------------
+  linear obs operator:
+    name: Composite
+    components:
+    - name: VertInterp
+      variables:
+      - name: windEastward
+      - name: windNorthward
+
+      # Hofx scaling
+      hofx scaling field: SurfaceWindScalingPressure
+      hofx scaling field group: DerivedVariables
+
+    - name: VertInterp
+      observation alias file: "/work2/noaa/da/acollard/git/GDASApp/parm/atm/obs/testing/swell/obsop_name_map.yaml"
```

Comment:
Should not need this, and it is pointed to Andrew's directory (probably copied over from GOES SWELL)